pkg/rbd/rbd_util.go

```diff
@@ -87,6 +87,11 @@ type rbdVolume struct {
 	DisableInUseChecks bool   `json:"disableInUseChecks"`
 	Encrypted          bool
 	KMS                util.EncryptionKMS
+
+	// connection
```

Comment:
if we keep this is Credentials structure, can we reuse the connection for rados and ceph fs also?

Reply:
Yes, it can be moved to the Credentials structure. I want to make changes like that in follow-up patches, so that there is no need to pass credentials around anymore.